src/D2L.Bmx/UpdateHandler.cs

```diff
@@ -64,8 +64,9 @@ internal class UpdateHandler( IGitHubClient github, IVersionProvider versionProv
 		string backupPath = Path.Join( workspaceDir, $"bmx-v{localVersion}-old.bak" );
 		try {
 			File.Move( currentFilePath, backupPath );
-		} catch( IOException ex ) {
-			throw new BmxException( "Could not remove the old version. Please try again with elevated permissions.", ex );
+		} catch( UnauthorizedAccessException ex ) {
+			throw new BmxException(
+				"Permission denied when removing the old version. Please try again with elevated permissions.", ex );
```

Comment:
@gord5500 I know you said you saw an IOException on Mac, but on WSL I'm seeing an `UnauthorizedAccessException` with `IOException` as an inner exception, and the old code wasn't catching the permission issue. Wonder if you misread the top level exception?
Can check again when you test it on Mac.

Reply:
`UnauthorizedAccessException` also just makes sense...

Reply:
I think you are right about that. I will make sure to check when I test

Reply:
Instead of saying "old version" (or in addition to), could we include the file path in the message? That way the user can just delete it in whichever way they see fit.

Reply:
Not a bad idea to include the file path for general troubleshooting. I'll update in another PR.